modules/registry-configuring-storage-azure-gov-cloud.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring_registry_storage-azure.adoc
+
+[id="registry-configuring-storage-azure-gov-cloud_{context}"]
+= Configuring registry storage for Azure Government Cloud
+
+During installation, your cloud credentials are sufficient to create Azure Blob
+Storage, and the Registry Operator automatically configures storage.
+
+.Prerequisites
+
+* A cluster on Azure with Government Cloud infrastructure.
+* To configure registry storage for Azure, provide Registry Operator
+cloud credentials.
+* For Azure storage the secret is expected to contain one key:
+** `REGISTRY_STORAGE_AZURE_ACCOUNTKEY`
+
+.Procedure
+
+. Create an link:https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal[Azure storage container].
+
+. Fill in the storage configuration in `configs.imageregistry.operator.openshift.io/cluster`:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry.operator.openshift.io/cluster
+----
++
+.Example configuration
+[source,yaml]
+----
+Storage:
+Azure:
+Account Name: imageregistryadcd
+Cloud Name: AzureUSGovernmentCloud
+Container: <container-name>
```

Comment:
```yaml
storage:
  azure:
    accountName: <storage-account-name>
    container: <container-name>
    cloudName: AzureUSGovernmentCloud
```

Reply:
btw, there are few more options: https://github.com/openshift/api/blob/43bf8c1cc69ff4f5468d22b5ffb241666aee77a8/config/v1/types_infrastructure.go#L300

* AzurePublicCloud (the default value if not set)
* AzureUSGovernmentCloud
* AzureChinaCloud
* AzureGermanCloud